### PR TITLE
Url update

### PR DESCRIPTION
**Pull request by felixleo22**

### CODE REVIEW

File: src/update/bin/steps.js [hunk 1]
```diff
@@ -452,8 +452,74 @@ const askUnpaywall = async (jobConfig) => {
...
+      writeStream.on('error', async (err) => {
+        logger.error(err);
+        await fail(stateName, err);
+        return reject(err);
+      });
+    });
+  } else {
+    const writeStream = await fs.createWriteStream(filepath);
+    writeStream.write(res.data);
+    writeStream.end();
```
Comment (nojhamster) [inline src/update/bin/steps.js:516]:
```suggestion
   try {
     await fs.writeFile(filepath, res.data);
   } catch (e) {
     await fail(stateName, err);
     return reject(e);
   }
```

Looks like you're just writing content to a file, so `fs.writeFile()` may be simpler. It's also easier to handle the error that way.

File: src/update/bin/state.js [hunk 2]
```diff
@@ -99,6 +99,8 @@ const addStepDownload = async (filename, downloadFile) => {
  * @param {String} downloadFile - unpaywall data update file name
  */
 const addStepInsert = async (filename, downloadFile) => {
+  console.log(filename);
+  console.log(downloadFile);
```
Comment (nojhamster) [inline src/update/bin/state.js:103]:
```suggestion
```

File: docker-compose.preprod.yml [hunk 3]
```diff
@@ -47,8 +47,8 @@ services:
       - EZUNPAYWALL_NOTIFICATIONS_RECEIVERS
       - EZUNPAYWALL_NOTIFICATIONS_MACHINE
        # unpaywall
-      - API_KEY_UPW
-      - UNPAYWALL_URL=http://api.unpaywall.org/feed/changefiles
+      - UNPAYWALL_APIKEY
+      - UNPAYWALL_HOST=http://api.unpaywall.org/feed/changefiles
```
Comment (nojhamster) [inline docker-compose.preprod.yml:51]: Why changing from `URL` to `HOST` ? `http://api.unpaywall.org/feed/changefiles` is a URL (where `api.unpaywall.org` is the host)

File: docker-compose.debug.yml [hunk 4]
```diff
@@ -36,8 +36,8 @@ services:
       - EZUNPAYWALL_NOTIFICATIONS_RECEIVERS
       - EZUNPAYWALL_NOTIFICATIONS_MACHINE=dev
       # unpaywall
-      - API_KEY_UPW=default
-      - UNPAYWALL_URL=http://fakeunpaywall:12000/feed/changefiles
+      - UNPAYWALL_APIKEY=default
+      - UNPAYWALL_HOST=http://fakeUnpaywall:12000
```
Comment (nojhamster) [inline docker-compose.debug.yml:40]: Why changing from `URL` to `HOST` ? `http://fakeUnpaywall:12000` looks like a URL (where `fakeUnpaywall:12000` is the host)

File: docker-compose.yml [hunk 5]
```diff
@@ -45,8 +45,8 @@ services:
       - EZUNPAYWALL_NOTIFICATIONS_RECEIVERS
       - EZUNPAYWALL_NOTIFICATIONS_MACHINE
        # unpaywall
-      - API_KEY_UPW
-      - UNPAYWALL_URL=http://api.unpaywall.org/feed/changefiles
+      - UNPAYWALL_APIKEY
+      - UNPAYWALL_HOST=http://api.unpaywall.org
```
Comment (nojhamster) [inline docker-compose.yml:49]: Why changing from `URL` to `HOST` ? `http://api.unpaywall.org` looks like a URL (where `api.unpaywall.org` is the host)

**Comment by felixleo22** [comment]
ok, i made the changes and the test past

**Pull request merged**